pkg/types/interfaces.go

```diff
@@ -52,3 +53,10 @@ type ObjectList interface {
 	ItemAt(index int) Object
 	Len() int
 }
+
+// ObjectPage is the DTO for a given page of resources when listing
```

Comment:
I think it would be useful to add to the doc that "_Token represents the base64 encoded paging_sequence of the last entity in items list_", as I think this is only mentioned in the documentation **.md** page and nowhere in the code.